index.html

```diff
@@ -436,13 +436,20 @@ <h3>Use Cases and Requirements</h3>
 <!-- requirement list start -->
         <ul>
           <li>
+          <li>
+<a>Credentials</a> represent statements made by an issuer.
+          </li>
 <a>Verifiable credentials</a> represent statements made by an <a>issuer</a> in
 a tamper-evident and privacy-respecting manner.
           </li>
           <li>
 <a>Holders</a> assemble collections of <a>verifiable credentials</a> from
 different <a>issuers</a> into a single artifact, a
-<a>verifiable presentation</a>.
+<a>presentation</a>.
+          </li>
+          <li>
+Holders convert <a>presentations</a> into <a>verifiable presentations</a>,
+rendering them tamper-evident. 
```

Comment:
```suggestion
Holders transform <a>presentations</a> into <a>verifiable presentations</a>,
in a way that renders them tamper-evident. 
```

Reply:
Very happy to accept your changes @msporny

Reply:
Thanks @David-Chadwick -- I prefer @dlongley's text, so removing my suggestion in favor of @dlongley's.